presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/source/BenchmarkSuiteDao.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark.source;
+
+import com.facebook.presto.benchmark.framework.BenchmarkQuery;
+import com.facebook.presto.benchmark.framework.BenchmarkSuiteInfo;
+import org.jdbi.v3.sqlobject.config.RegisterColumnMapper;
+import org.jdbi.v3.sqlobject.config.RegisterConstructorMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.customizer.Define;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+import java.util.List;
+
+@RegisterColumnMapper(StringToStringMapColumnMapper.class)
+@RegisterColumnMapper(PhaseSpecificationsColumnMapper.class)
+public interface BenchmarkSuiteDao
+{
+    @SqlUpdate("CREATE TABLE benchmark_suites (\n" +
+            "  `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n" +
+            "  `suite` varchar(256) NOT NULL,\n" +
+            "  `query_set` varchar(256) NOT NULL,\n" +
+            "  `phases` mediumtext NOT NULL,\n" +
+            "  `session_properties` mediumtext NOT NULL,\n" +
+            "  `created_by` varchar(256) NOT NULL,\n" +
+            "  `create_time` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,\n" +
+            "  `update_time` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,\n" +
+            "  PRIMARY KEY (`id`),\n" +
+            "  UNIQUE KEY suite (`suite`))")
+    void createBenchmarkSuitesTable();
+
+    @SqlUpdate("CREATE TABLE benchmark_queries (\n" +
+            "  `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n" +
```

Comment:
In general I don't see the point of having unsigned ID while they will be signed numbers in code. Though from the database table's perspective unsigned makes sense. Not likely to cause real trouble though.